ui-tests/src/test/java/org/openmrs/reference/page/ManageHtmlFormsPage.java

```diff
@@ -0,0 +1,24 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
```

Comment:
nit: since this is a new file, it should be indented with tabs, not spaces.

Reply:
Resolved thanks